cache.go

```diff
@@ -101,35 +120,35 @@ func (cacher *Cacher) TryGetInfrastructureSchema(alias string) (*InfrastructureR
 }
 
 func (cacher *Cacher) doCacheTiers(client *Client) {
-	log.Debug().Msg("Caching 'Tier' lookup table from API ...")
+	cacher.log.Debug().Msg("Caching 'Tier' lookup table from API ...")
 
 	data, dataErr := client.ListTiers()
 	if dataErr != nil {
-		log.Warn().Msgf("===> Failed to list all 'Tier' from API - REASON: %s", dataErr.Error())
```

Comment:
Changed these lines to use `.Err()` since it's convenient and makes the logs more structured.

Using `dataErr.Error()` and `%s` is not necessary, errors can be printed out using `%w`.

```
log.Warn().Msgf("===> Failed to list all 'Tier' from API - REASON: %w", dataErr)
```